src/mocks/handlers.ts

```diff
@@ -0,0 +1,139 @@
+import { rest } from "msw";
+
+import { STELLAR_EXPERT_URL } from "constants/apiUrls";
```

Comment:
We could name this `STELLAR_EXPERT_AMM_URL` to not confuse it with `https://stellar.expert/explorer`.